### ISSUE TITLE
Settings option to remove the Keywords section and Keyword Count

### ISSUE THREAD

**Issue by itsmerobliberty** [Sterc/SEOPro#71]
We love the Google Preview and word count, but the Keywords bit is very confusing to some clients. It would be awesome to have an option to remove that.

**Comment by himurovich** [comment]
Agree, as we know (and heard again together with Sterc from Steven J McLean at MODXPO'17 :) ) today this is insignificant meta field. We really need some system option, keywords field should be optional.

**Comment by Erdem-Ozyalcin** [comment]
I am also using SEOPro and I also wil love to hide the Keywords Field. It is very confusing if you don’t use it.
